### PR TITLE
Docs update Schedules concepts page

### PR DESCRIPTION
**Pull request by discdiver**
### Summary
Update Schedules concepts pages to show how to use the UI and deployment YAML file

This pull request is:

- [x] A documentation / typographical error fix
	- No tests or issue needed
- [ ] A short code fix
	- Please reference the related issue by including "closes `<link to issue>`" in this Pull Request's summary section. 
        - If no issue exists, please create a bug report issue 
	- Please include tests. One-line fixes without tests will not be accepted unless it's related to the documentation only.
- [ ] A new feature implementation
	- Please reference the related issue by including "closes `<link to issue>`" in this Pull Request's summary section. 
        - If no issue exists, please create a feature enhancement issue 
	- Please include tests
    - Please make sure that your QA steps are both thorough and easy to reproduce by somebody with limited knowledge of the feature that you are submitting

**Happy engineering!**

### CODE REVIEW

File: docs/concepts/schedules.md [hunk 1]
```diff
@@ -16,63 +16,48 @@ Schedules tell the Prefect API how to create new flow runs for you automatically
...
+- [`Interval`](#interval) is best suited for deployments that need to run at some consistent cadence that isn't related to absolute time.
+- [`RRule`](#rrules) is best suited for deployments that rely on calendar logic for simple recurring schedules, irregular intervals, exclusions, or day-of-month adjustments.
 
-To remove all scheduled runs for a flow deployment, update the deployment with no `schedule` parameter.
 
-## CronSchedule
+## Creating schedules through the deployment YAML file
 
-A `CronSchedule` creates new flow runs according to a provided [`cron`](https://en.wikipedia.org/wiki/Cron) string. Users may also provide a timezone to enforce DST behaviors.
+You create a schedule by including a `schedule` section as part of the [deployment YAML file](/concepts/deployments/#deploymentyaml) for a deployment. Here's an example that uses cron and specifies a timezone.
```
Comment (tpdorsey) [inline docs/concepts/schedules.md:44]:
```suggestion
You may create a schedule by including a `schedule` section as part of the [deployment YAML file](/concepts/deployments/#deploymentyaml) for a deployment. Here's an example that uses cron and specifies a timezone. 
```

File: docs/concepts/schedules.md [hunk 2]
```diff
@@ -16,63 +16,48 @@ Schedules tell the Prefect API how to create new flow runs for you automatically
...
 
-Deployment(
-    name="cron-schedule-deployment",
-    flow_location="/path/to/flow.py",
-    schedule=CronSchedule(
-        cron="0 0 * * *",
-        timezone="America/New_York"),
-)
-```
+A `CronSchedule` creates new flow runs according to a provided [`cron`](https://en.wikipedia.org/wiki/Cron) pattern. Users may also provide a timezone to enforce DST behaviors.
```
Comment (tpdorsey) [inline docs/concepts/schedules.md:56]: Suggest linking to the API definition for each of these. It'll make clear the available options.

**Review by tpdorsey — APPROVED** [review]
Minor suggestions. Looks good.

File: docs/concepts/schedules.md (hunk 2 above)
Comment (discdiver) [reply docs/concepts/schedules.md]: I Is it cool to leave out the API definition for now @tpdorsey ? I feel like it's a little confusing. Thinking we add back in with any Deployment updates. I removed mention of the Python classes in the changes I just pushed now.

Comment (tpdorsey) [reply docs/concepts/schedules.md]: yep let's revisit later

**Pull request merged**